### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by chrisjwaddell** [LiveReader/graphly-d3#85] [Ideas]
Graphly looks great. I would love to see a couple more image examples on the documentation to get a quick snapshot of what it can do. That would help to get a very quick idea of what you can do with it. Pictures are good. A picture tells a thousand words.

**Comment by jason-rietzke** [comment]
Glad to hear you like Graphly 🙂
You are absolutely right, there should be more use cases accessible to see what it can be used for. In fact, we are currently working on a CDN service for GraphlyD3 Templates and this would be a great foundation to include interactive demos of already existing visualizations 👍

**Reply by jason-rietzke** [reply]
FYI we recently made [Graphly Hub](https://graphly.dev/) accessible to the public. Right now accounts are in closed beta but every Template available is visible and you can even interact with it by changing the payload to get a grasp of the functionalities of each shape.
